.gitignore

```diff
@@ -2,7 +2,6 @@
 .idea/
 
 node_modules/
-package-lock.json
```

Comment:
It's still not obvious to me that we want this file. What's your reasoning?

Reply:
Well I believe it is standard industry practise, but if you are dead against it, I can go back to ignoring it again...

Reply:
My worry is that it will prevent CI from pulling the most recent dependencies in favour of faithfully reproducing what the last developer was using. i.e. It's good for the developer experience but bad for security.

Reply:
If by "developer experience" you mean correctly functioning software, sure... but fine, I'll remove it...